spec/features/projects_manage_spec.rb

```diff
@@ -125,26 +127,12 @@
       find("#import-export").click
       page.attach_file("file", (Rails.root + "spec/fixtures/test.csv").to_s)
       click_on "Import"
-      expect(project.stories.count).to be > 1
+      expect(page).to have_no_content "Import CSV"
+      expect(page).to have_content "CSV import was successful"
+      expect(project.stories.count).to be 9
       expect(project.stories.map(&:title).join).to include("php upgrade")
-      expect(page.text).to include("success")
       expect(page.current_path).to eql project_path(project.id)
     end
-
-    it "allows me to update existing stories on import" do
```

Comment:
This flaky test was failing on Gelsey's machine. For that reason we decided remove it